### PR TITLE
Date Range Selection (PT-186719476)

### PR DESCRIPTION
**Pull request by lublagg**
This PR adds all of the date range functionality.
It also adds the warning modal if the user has selected a frequency + date range combination that produces more than 5,000 entries. I didn't realize that the warning modal had already been added at the `App` component level previously so I did some duplicate work. But I think it's better handled by the `date-range` component because it's my understanding that the modal should pop up after a user has selected the dates, not when they click the "Get Data" button.

### CODE REVIEW

File: src/components/App.tsx [hunk 1]
```diff
@@ -111,12 +44,11 @@ export const App = () => {
       <div className="divider" />
       <AttributesSelector />
       <div className="divider" />
-      <div className="footer">
+      <div className="fooster">
```
Comment (eireland) [inline src/components/App.tsx]: Any reason for the change in spelling?

File: src/components/date-range/calendars.scss [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  flex-direction: column;
+  align-self: center;
+  justify-content: center;
+  align-items: center;
+  border-radius: 8px;
+  box-shadow: 0 1px 10px 0 rgba(0, 0, 0, 0.25);
+  .calendar-container {
+    display: flex;
+    width: 342px;
+    height: 224;
```
Comment (eireland) [inline src/components/date-range/calendars.scss]:
```suggestion
    height: 224px;
```

File: src/components/date-range/calendars.scss [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    background-color: white;
+    border-radius: 8;
```
Comment (eireland) [inline src/components/date-range/calendars.scss]:
```suggestion
    border-radius: 8px;
```

File: src/components/date-range/calendars.scss [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    .calendar-column {
+      width: 171px;
+      height: 224;
```
Comment (eireland) [inline src/components/date-range/calendars.scss]:
```suggestion
      height: 224px;
```

File: src/components/date-range/calendars.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import "./calendars.scss";
+import { Calendar } from "./calendar";
+import { useStateContext } from "../../hooks/use-state";
+
+interface ICalendarsProps {
+  selectedCalendar: string|undefined;
```
Comment (eireland) [inline src/components/date-range/calendars.tsx]: Just for consistency
```suggestion
  selectedCalendar: string | undefined;
```

**Review by eireland — APPROVED** [review]
Just some linting corrections
Otherwise LGTM

File: src/components/App.tsx (hunk 1 above)
Comment (lublagg) [reply src/components/App.tsx]: I have no idea why that happened!

**Pull request merged**